src/operators.jl

```diff
@@ -191,7 +191,7 @@ function embed(basis_l::CompositeBasis, basis_r::CompositeBasis,
     data[1] = one(Tnum)
     i = N
     while i > 0
-        if i ∈ index
+        if i == index
```

Comment:
this seems to have been an old copy-paste "mistake" when the multi-index embed was being turned into a single-index embed fast path. It does not matter because `::Integer ∈ ::Integer` is the same as `::Integer == ::Integer`